### PR TITLE
Add template metaflow flows!

### PR DESCRIPTION
**Pull request by chelseatroy**
### Goal(s) of this Pull Request:

- We want to make it easy for our clients (Machine Learning Engineers on subject matter teams at Mozilla) to quickly productionize their models using the tools we've (painstakingly) prepared for them. This example flow provides them a starting point with:

1. An existing `FlowSpec` for using the Metaflow API for model orchestration
2. Existing integration with Weights & Biases for experiment analysis
3. Examples of the pieces many MLE teams will need, like including config files, running on docker containers, and logging events.

### Example of how it works:

Out of the box, the flow should "work"—that is, doing `python template_flow.py show` should give an outline of the steps and their descriptions, and `python template_flow.py run` should run the flow on our production outerbounds instance (provided that the client is set up to use Outerbounds—README for that in the next PR).

### Acceptance criteria:

In order to approve, reviewer must be able to...

- [ ] `cd` into the folder with `template_flow.py`, run `python template_flow.py show`, and get an outline of the steps
- [ ] `cd` into the folder with `template_flow.py` (having already been set up with the Mozilla Outerbounds instance), run `python template_flow.py run`, and have the flow run on our production Outerbounds instance.

### CODE REVIEW

File: src/mozmlops/templates/template_flow.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import os
+
+from metaflow import (
+    FlowSpec, IncludeFile, Parameter,
+    card, current, step,
+    environment, kubernetes
+)
+from metaflow.cards import Markdown
+
+from artifact_store import ArtifactStore
```
Comment (Dexterp37) [inline src/mozmlops/templates/template_flow.py:14]: The import is no longer in this directory, so this statement will make the flow fail

**Review by Dexterp37 — CHANGES_REQUESTED** [review]

File: src/mozmlops/templates/template_flow.py (hunk 1 above)
Comment (chelseatroy) [reply src/mozmlops/templates/template_flow.py:14]: Good catch. I haven't tried to run these flows from here yet. I'll do that today.

Comment (chelseatroy) [reply src/mozmlops/templates/template_flow.py:14]: OK. Fixed. Ran the `show` command to confirm that the change works!

Comment (Dexterp37) [reply src/mozmlops/templates/template_flow.py:14]: I believe the fix was mistakenly committed to #4 . I'm fine with merging this in the broken state, and take further action in #4

**Review by Dexterp37 — APPROVED** [review]

**Pull request merged**